OtripleS.Web.Api/Services/CourseAttachments/CourseAttachmentService.cs

```diff
@@ -0,0 +1,45 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using OtripleS.Web.Api.Brokers.DateTimes;
+using OtripleS.Web.Api.Brokers.Loggings;
+using OtripleS.Web.Api.Brokers.Storage;
+using OtripleS.Web.Api.Models.CourseAttachments;
+
+namespace OtripleS.Web.Api.Services.CourseAttachments
+{
+    public partial class CourseAttachmentService : ICourseAttachmentService
+    {
+
+        private readonly IStorageBroker storageBroker;
+        private readonly ILoggingBroker loggingBroker;
+        private readonly IDateTimeBroker dateTimeBroker;
+
+        public CourseAttachmentService(
+            IStorageBroker storageBroker,
+            ILoggingBroker loggingBroker,
```

Comment:
Date brokers should come before logging brokers in here .
Because datetime brokers are more important to any given operation to happen - logging brokers are just helpers